common/components/common/projects/AboutProjectDisplay.jsx

```diff
@@ -108,8 +108,9 @@ class AboutProjectDisplay extends React.PureComponent<Props, State> {
 
   render(): React$Node {
     const project = this.state.project;
+    const widthModifier=window.parent !== window ? ' use-parent-width' : '';
```

Comment:
The `window.parent !== window` check is made in a lot of places, it should live in a helper function.

Reply:
isWithinIframe - done.